### PR TITLE
add js-viewer module

### PR DESCRIPTION
**Pull request by leminam**

### CODE REVIEW

File: src/modules/portfolio/portfolio-item.hbs [hunk 1]
```diff
@@ -8,22 +8,22 @@
     <div class="portfolio__info-description">{{ info_description }}</div>
   </div>  
   <div class="portfolio__devices">
-    <div class="portfolio__devices-nav">
+    <div class="JS-Viewer portfolio__devices-nav">
       <span class="portfolio__devices-item">
-        <a href="#" class="portfolio__devices-link">Desktop</a>
+        <a href="javascript:;" class="JS-View-Type portfolio__devices-link">Desktop</a>
```
Comment (lemind) [inline src/modules/portfolio/portfolio-item.hbs]: like common class name = JS-Viewer

File: src/static/js/components/viewer.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  construct() {
+    this.elType = this.elem.getElementsByClassName('JS-View-Type')[0];
+    this.elImage = this.elem.getElementsByClassName('JS-View-Image')[0];
+
+    this.init();
+    this.ready();
+  }
+
+  init() {
+    var _this = this;
```
Comment (lemind) [inline src/static/js/components/viewer.js:41]: add console.log(this.elType) here and look what is going on.

File: src/static/js/components/viewer.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+export default class Viewer {
+  constructor(elem, params = {}) {
+    this.elem = elem;
+    this.params = params;
+
+    this.cssElemType = this.params.cssElemType || '';
+    this.cssElemImage = this.params.cssElemImage || '';
```
Comment (lemind) [inline src/static/js/components/viewer.js]: add console.log(this.cssElemImage) and check what is going on here, what inside this.cssElemImage ?

File: src/static/js/init/main.js [hunk 4]
```diff
@@ -1,2 +1,62 @@
...
+    return false;
+  }
+
+  const common = {};
+
+  Array.prototype.forEach.call(document.getElementsByClassName('JS-Viewer'), function(elem) {
+    if (!elem.classList.contains("JS-Viewer-Ready")) {
+      const elParams = GLOBAL.parseData(elem.dataset.viewerParams);
+      const params = Object.assign({}, common, elParams);
+
```
Comment (lemind) [inline src/static/js/init/main.js]: add console.log(Viewer)
add console.log(elem)
add console.log(params)

what inside them?

File: src/modules/portfolio/portfolio-item.hbs (hunk 1 above)
Comment (leminam) [reply src/modules/portfolio/portfolio-item.hbs]: fixed

File: src/static/js/components/viewer.js (hunk 2 above)
Comment (leminam) [reply src/static/js/components/viewer.js:41]: https://nimb.ws/PB4bxX - 3 span with JS clases

File: src/static/js/components/viewer.js (hunk 3 above)
Comment (leminam) [reply src/static/js/components/viewer.js]: fixed

File: src/static/js/init/main.js (hunk 4 above)
Comment (leminam) [reply src/static/js/init/main.js]: console.log(Viewer) - shows the function, some details
console.log(elem) - shows 3 div with JS clases
console.log(params) - shows 3 objects {}

**Pull request merged**